src/main/resources/assets/paths/studyById.yaml

```diff
@@ -21,4 +21,51 @@ get:
     404:
       description: Not Found
     500:
-      description: Server error
\ No newline at end of file
+      description: Server error
+put:
+  summary: Update Study by Id
+  description: Updates Study and Properties of the Study specified by Id.
+  tags:
+    - Dataset
+    - Study
+  parameters:
+    - name: studyId
+      in: path
+      description: ID of the Study
+      required: true
+      schema:
+        type: integer
+  requestBody:
+    description: |
+      Dataset registration schema instance and optional file for
+      an alternative data sharing plan. The registration schema is 
+      always required. If a file is also provided, then alternative 
+      data sharing information is required in the registration schema.
+    content:
+      multipart/form-data:
+        schema:
+          type: object
+          properties:
+            dataset:
+              description: Required dataset registration json schema instance
+              type: object
+              $ref: '../schemas/DatasetRegistrationSchemaV1.yaml'
```

Comment:
Technically, `$ref` is not allowed here but this does work in practice and is necessary to fully document this API